enos/modules/local_kind_cluster/main.tf

```diff
@@ -0,0 +1,50 @@
+terraform {
+  required_providers {
+    enos = {
+      source = "app.terraform.io/hashicorp-qti/enos"
+    }
+    random = {
+      source  = "hashicorp/random"
+      version = ">= 3.4.3"
+    }
+  }
+}
+
+resource "random_pet" "cluster_name" {}
```

Comment:
lolwut, random pet?

Reply:
yeah it's an awesome resource... my go to when I need a name for something and don't want just a random sequence of letters. This resource creates a hyphenated name from a selection of a bunch of words and names.